formatters/html/html.go

```diff
@@ -435,15 +443,15 @@ func (f *Formatter) styleToCSS(style *chroma.Style) map[chroma.TokenType]string
 		if t != chroma.Background {
 			entry = entry.Sub(bg)
 		}
-		if !f.allClasses && entry.IsZero() {
+		if !f.allClasses && entry.IsZero() && f.customCSS[t] == `` {
```

Comment:
I think this should probably search through the token hierarchy, ie. t.SubCategory() and t.Category(). This will let allow the CSS to be applied to an entire category/subcategory of tokens easily, eg. `Text` or `Comment`.

Reply:
Is the new commit OK?

Reply:
LGTM!